server/services/applicationService.ts

```diff
@@ -62,8 +62,8 @@ export default class ApplicationService {
       const oldBody = application.data?.[taskName]?.[pageName]
 
       application.data = this.addPageDataToApplicationData(application.data, taskName, pageName, page)
-      application.data = this.removeConditionalData(application.data)
-      application.data = this.removeCheckYourAnswersIfPageChange(application.data, pageName, oldBody, page.body)
+      application.data = this.deleteOrphanedFollowOnAnswers(application.data)
```

Comment:
change approved in this PR but I forgot to push changes https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/284